engine/runtime/src/main/scala/org/enso/compiler/pass/analyse/AliasAnalysis.scala

```diff
@@ -414,9 +415,10 @@ case object AliasAnalysis extends IRPass {
         }
 
         val labelId = graph.nextId()
-        parentScope.add(
+        val definition =
           Occurrence.Def(labelId, label.name, label.getId, label.getExternalId)
-        )
+        parentScope.add(definition)
```

Comment:
Since this is repeated, maybe worth having `addWithDefintion` method in `Scope` that does `add` and `addDefinition`?

Reply:
Probably. But there is also a call site that calls only `addDefinition` and not `add`, which is vital. So instead of having `add`, `addDefinition` and `addWithDefinition`, I will rather repeat `add` and `addDefinition` on few places.